doc/tutorials/enable_ivshmem.rst

```diff
@@ -153,7 +153,7 @@ Linux-based post-launched VMs (VM1 and VM2).
 
         acrn-dm -A -m $mem_size -s 0:0,hostbridge \
          -s 3,virtio-blk,/home/acrn/UserVM2.img \
-         -s 4,virtio-net,tap0 \
+         -s 4,virtio-net,tap=0 \
```

Comment:
```suggestion
         -s 4,virtio-net,tap=tap0 \
```

Reply:
As previous.
Could we use tap=LaaG to show the virtio-net format?